src/app/home/home.component.html

```diff
@@ -1 +1,50 @@
-<h1>Welcome to Miro!</h1>
+<div class="jumbotron" style="background-image: url('https://wallpapershome.com/images/pages/pic_h/19468.jpg');
```

Comment:
This image should be downloaded and served locally in case it is removed from the remote site in the future. That said, the licensing on this image is not explicit, as per what I could find at https://wallpapershome.com/art/anime-girl-beauty-4k-19468.html